test/unit-tests/components/views/rooms/RoomListPanel/RoomListOptionsMenu-test.tsx

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2025 New Vector Ltd.
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only OR GPL-3.0-only OR LicenseRef-Element-Commercial
+ * Please see LICENSE files in the repository root for full details.
+ */
+
+import React from "react";
+import { render, type RenderOptions, screen } from "jest-matrix-react";
+import userEvent from "@testing-library/user-event";
+import { TooltipProvider } from "@vector-im/compound-web";
+
+import { RoomListOptionsMenu } from "../../../../../../src/components/views/rooms/RoomListPanel/RoomListOptionsMenu";
+import { type RoomListViewState } from "../../../../../../src/components/viewmodels/roomlist/RoomListViewModel";
+
+function getRenderOptions(): RenderOptions {
+    return {
+        wrapper: ({ children }) => <TooltipProvider>{children}</TooltipProvider>,
```

Comment:
Do we need it?

We don't use tooltip in the menu and I think that `jest-matrix-react` render already adds the tootltip provider

Reply:
Good point, we don't, I think I forgot to use jest-matrix-react at one point.